docs/breadcrumbs/mass-deployment.md

```diff
@@ -0,0 +1,11 @@
+# Mass Deployment
+
+Breadcrumbs are volatile and take slightly more upkeep than Tokens and Canaries themselves. They are relatively short-lived, lasting only until the Canary's settings (or IP) change to invalidate them. This means that when there are Canary changes, they may need to be freshly re-deployed.
```

Comment:
> take slightly more upkeep

Maybe rather "need slightly more upkeep"?

> until the Canary's settings (or IP) change to invalidate them. This means that when there are Canary changes, they may need to be freshly re-deployed.

I think "change" needs to be "changes" here and that "freshly redeployed" should be either just "redeployed" or "freshly deployed". Although that could make it sound that the same breadcrumb needs to be redployed instead of a new one having to be generated and deployed.

"until a Canary's settings (or IP) changes. Depending on the what was changed, they may need to be freshly deployed/redeployed."

Reply:
Thanks Tom!